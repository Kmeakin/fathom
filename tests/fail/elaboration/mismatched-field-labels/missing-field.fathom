//~ exit-code = 1

{ x = {} } : { x : Type, y : Type }
