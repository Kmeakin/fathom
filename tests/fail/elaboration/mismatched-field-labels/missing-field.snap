stdout = ''
stderr = '''
error: mismatched field labels in record literal
  ┌─ tests/fail/elaboration/mismatched-field-labels/missing-field.fathom:3:1
  │
3 │ { x = {} } : { x : Type, y : Type }
  │ ^^^^^^^^^^ missing fields `y`
  │
  = expected fields `x`, `y`
  =    found fields `x`

'''
